src/test/resources/journeys.conf

```diff
@@ -15,52 +15,26 @@
 # Configure here your journeys. A journey is a sequence of requests at a certain load.
 
 journeys {
-
-  individual-journey = {
-    description = "Individual journey"
-    load = 9.1
-    parts = [
-        DigitalDisclosureIndividualJourney
-    ]
-  }
-
-  estate-journey = {
-    description = "Estate Journey"
-    load = 9.1
+  disclosure-company = {
```

Comment:
I've defined **ONE** "happy path" journey here.

The journey defined here should really be the one with the most anticipated traffic for the service. For example, a previous project I worked on expected 90%+ journeys to be Limited Company Registration, so that's the journey we focused on for our performance tests.

Please confirm the happy path that the majority of users are expected to complete and target this.